### PR TITLE
A dice rolling simulation script

### PR DESCRIPTION
**Pull request by SlimxShadyx**
A beginner project for people wanting an example of how you can use while loops, and random.

Key parts of the code is commented, may comment more later to help explain.

I am a little new to Python myself, so please feel free to let me know of any better techniques I can use!

### CODE REVIEW

File: dice_rolling_simulator.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+#These variables are used for user input and while loop checking.
+correct_word = False
+dice_checker = False
+dicer = False
+roller_loop = False
+
+#Checking the user input to start the program.
+while correct_word == False:
+
+    user_input_raw = raw_input("\r\nWelcome to the Dice Rolling Simulator! We currently support 6, 8, and 12 sided die! Type [start] to begin!\r\n?>")
```
Comment (mam8) [inline dice_rolling_simulator.py]: The prompt string is too long for one line. You may have a variable named prompt_str, and use () to limit characters number.

File: dice_rolling_simulator.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+        elif user_dice_chooser == 8:
+            dice_checker = True
+
+        elif user_dice_chooser == 12:
+            dice_checker = True
+
+        else:
+            print "\r\nPlease choose one of the applicable options!\r\n"
+    
+    #Another inner while loop. This one does the actual rolling, as well as allowing the user to re-roll without restarting the program.
```
Comment (mam8) [inline dice_rolling_simulator.py]: separate  the comment to multiple lines because it's too long.

File: dice_rolling_simulator.py [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    while dicer == False:
+
+        if user_dice_chooser == 6:
+            dice_6 = random.randint(1,6)
```
Comment (mam8) [inline dice_rolling_simulator.py]: It seems there are too many duplicated codes, could you abstract them (line 68~ 79, line 53~64, etc) to a function.

File: dice_rolling_simulator.py (hunk 1 above)
Comment (Angus3321) [inline dice_rolling_simulator.py]: Sooo dramtic the words you uesd!!

File: dice_rolling_simulator.py [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+            user_exit_checker = (user_exit_checker_raw.lower())
+
+            if user_exit_checker == 'roll':
+                dicer = False
+
+            elif user_exit_checker == 'exit':
+                roller_loop = True
+
+print "Thanks for using the Dice Rolling Simulator! Have a great day! =)"
+
```
Comment (Angus3321) [inline dice_rolling_simulator.py]: all right.Good job! but too many dulplicated contents you know,like another reviewer said.

**Comment by geekcomputers** [comment]
Thank you for you contribution to the repo and the community

**Comment by SlimxShadyx** [comment]
Thanks for merging and all the tips! :)

**Pull request merged**